### PR TITLE
change sigmoid and tanh to torch from nn.functional

### PR DESCRIPTION
**Pull request by dhpollack**
I received the following message when running `python check.py`

```
miniconda3/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
miniconda3/lib/python3.6/site-packages/torch/nn/functional.py:995: UserWarning: nn.functional.tanh is deprecated. Use torch.tanh instead.
  warnings.warn("nn.functional.tanh is deprecated. Use torch.tanh instead.")
Forward: Baseline (Python) vs. C++ ... Ok
```

afterwards I get

```
Forward: Baseline (Python) vs. C++ ... Ok
```

### CODE REVIEW

**Comment by goldsborough** [comment]
Thanks!
